hydrastart-postgres.yml

```diff
@@ -0,0 +1,52 @@
+###########################################################################
+#######             FOR DEMONSTRATION PURPOSES ONLY                 #######
+###########################################################################
+#                                                                         #
+# If you have not yet read the tutorial, do so now:                       #
+#  https://www.ory.sh/docs/hydra/5min-tutorial                            #
+#                                                                         #
+# This set up is only for demonstration purposes. The login               #
+# endpoint can only be used if you follow the steps in the tutorial.      #
+#                                                                         #
+###########################################################################
+
+version: '3'
```

Comment:
Would it be possible to use the `quickstart` yaml files as a basis for this exercise? You can override values of other docker-compose definition files which makes this highly reusable. For example, you would not need to define hydra-migrate or hydra at all!